waflyctl.go

```diff
@@ -535,6 +535,7 @@ func createOWASP(client fastly.Client, serviceID string, config TOMLConfig, wafI
 // DeleteLogsCall removes logging endpoints
 func DeleteLogsCall(client fastly.Client, serviceID string, config TOMLConfig, version int) bool {
 
+	Info.Printf("Deleting Web logging endpoint: '%s'", config.Weblog.Name)
```

Comment:
just use %q?

Reply:
Use of `%q` and `\n` seems pretty inconsistent across the whole.
I will make some changes and use %q, but didn't want to embark on changes across the whole thing in this PR.
Newlines are inconsequential because they're added by the logger if they're not there. But for consistency, I agree we should probably be consistent. Happy to do another commit either removing or adding. Let me know your preference?

Reply:
Happy for you to address it separately.